scripts/_helpers.py

```diff
@@ -27,6 +27,28 @@
 REGIONS_CONFIG = "regions_definition_config.yaml"
 
 
+def check_config_version(config):
+    """
+    Check that a version of the local config.yaml matches to the actual config
+    version as defined in config.default.yaml.
+    """
+
+    # using snakemake capabilities to deal with yanl configs
+    with open("config.default.yaml", "r") as f:
+        actual_config = yaml.safe_load(f)
+    actual_config_version = actual_config.get("version")
```

Comment:
I'd propose to have the file path as parameter here, like:

```suggestion
def check_config_version(config, fp_config="config.default.yaml"):
    """
    Check that a version of the local config.yaml matches to the actual config
    version as defined in config.default.yaml.
    """

    # using snakemake capabilities to deal with yanl configs
    with open(fp_config, "r") as f:
        actual_config = yaml.safe_load(f)
    actual_config_version = actual_config.get("version")
```

Reply:
Agree, done!